src/main/java/org/broadinstitute/hellbender/tools/CountBases.java

```diff
@@ -35,13 +37,17 @@
 
     private long count = 0;
 
+    @ArgumentCollection
+    final public SimpleOutputCollection out = new SimpleOutputCollection();
```

Comment:
Update the docs for this tool to mention that in addition to writing to stdout, it can also optionally output to a local or remote file via `-O` (this comment also applies to the other tools you modified below).